backend/api/main.py

```diff
@@ -1,14 +1,30 @@
 import uvicorn
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 from schemas import AssignmentMatrixInfo
 from schemas import SolutionRead
 from services import solve_problem
 
 app = FastAPI(title="Hungarian Algorithm API", version="0.1.0")
 
 
+app.add_middleware(
```

Comment:
Because of this the way I have suffered 😪 .
Nice work @Rexosei111

Reply:
Aw sorry🥲